benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/utils/load/mysql/MysqlTpchSchema.java

```diff
@@ -0,0 +1,132 @@
+package com.amazon.opendistroforelasticsearch.sql.benchmark.utils.load.mysql;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+public class MysqlTpchSchema {
+
+  public static String databaseName = "benchmark";
+  public static Map<String, List<String>> primaryKeyMap;
+  public static Map<String, Map<String, Map<String, String>>> foreignKeyMap;
+  public static Map<String, Map<String, String>> schemaMap;
+
+  static {
+    primaryKeyMap = new LinkedHashMap<>();
+    primaryKeyMap.put("customer", new LinkedList<>(Arrays.asList("c_custkey")));
```

Comment:
Why LinkedList instead of ArrayList?

Reply:
No specific reason. We can change this if needed. It's used in the load and query package. It was easier for me to understand problems when I maintained order.

Reply:
Maintaining order is imp here because tables with foreign key references have to be created first. otherwise, MySQL will throw an exception as `Failed to open the referenced table`